.gitignore

```diff
@@ -15,39 +15,30 @@
 *.idea
 *.iml
 
-# External dependencies
-
 # Build output
 target
 dist
 
 # Mac OS X Finder
 .DS_Store
 
-# Closed source libraries
-closed-lib
-
 # Node, Bower dependencies
 node_modules
 bower_components
 
-# Protractor logs
-protractor/logs
-
 # npm-debug log
 npm-debug.log
 
 # karma reports
 report.*.json
 
-# Lighthouse reports
-.lighthouseci
-
 # e2e test artifacts
 test-results
-allure-results
 
-package-lock.json
-
-#codecov artifacts
+# codecov artifacts
+.nyc_output
+coverage
 codecov
+
+# :(
+package-lock.json
```

Comment:
It doesn't matter in the context of this review, but I'm curious as to why we don't use `package-lock`?

Reply:
![thousandyardstare](https://user-images.githubusercontent.com/12446869/176252218-5eac5528-3289-4d8b-802b-31753c8a50fc.gif)